camunda/v2/client.go

```diff
@@ -153,3 +154,57 @@ func (c *client) doPostRequest(ctx context.Context, params *bytes.Buffer, endpoi
 
 	return body, nil
 }
+
+func (c *client) DeleteTask(ctx context.Context, businessKey string) error {
```

Comment:
this method (`DeleteTask`) is not totally clear to me, here we are not deleting any task, or the task is deleted when the process instance is deleted?

also the `businessKey` is not a task id because the method fails in case of multiple occurrence... 
then maybe we can refactor the name to something more explicit like:

```suggestion
// DeleteSingleTaskByBusinessKey will delete the task linked to the provided businessKey.
// An error will be returned in case of cero or  multiple tasks are related to the same businessKey.
func (c *client) DeleteSingleTaskByBusinessKey(ctx context.Context, businessKey string) error {
```

also will be nice to have some docs explain

Reply:
as discussed we change the name to **DeleteTaskByBusinessKey**